source/chapters/effects.rst

```diff
@@ -40,9 +40,20 @@ are located in the deck, microphone, and auxiliary sections of the skin. The
 buttons for routing other inputs to effect units are in different locations
 depending on the skin.
 
+.. figure:: ../_static/effect_unit_buttons.png
+   :align: center
+   :alt: Effect unit buttons located in the deck
+   :figclass: pretty-figures
+
+   The effect unit buttons located in the deck.
+
 If a channel is assigned to multiple effect units, those effect units are
 chained together. This allows you to chain more than 3 effects at a time.
 
+To route deck 1 to effect unit 1, click the :guilabel:`FX1` button on deck 1 to turn it on.
+To route deck 2 to effect unit 2, click the :guilabel:`FX2` button on deck 2, and so on.
+When routing decks to effect units, careful consideration should be given to which decks the effect units are being routed to.
```

Comment:
That sentence is a bit superfluous IMHO. What about:

```suggestion
It's also possible to route a single deck to multiple effect units, or multiple decks to a single effect unit.
```

Reply:
ping @deborahtrez